CHANGELOG.md

```diff
@@ -72,6 +72,7 @@ Read more about the session argument for computed fields in the [docs](https://h
 - server: fix importing of allow list query from metadata (fix #4687)
 - server: flush log buffer during shutdown (#4800)
 - server: fix edge case with printing logs on startup failure (fix #4772)
+- server: add flags for HASURA_GRAPHQL_EVENTS_HTTP_POOL_SIZE, HASURA_GRAPHQL_EVENTS_FETCH_INTERVAL and add a new config environment variable HASURA_GRAPHQL_LOG_HEADERS_FROM_ENV and a corresonding flag to it.
```

Comment:
Can we give the names for the flags instead?

Something like "Added startup flags --a , --b , --c"

Reply:
I'll update it.